docs/configuration/schema-config-reference.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Schema Configuration"
+linkTitle: "Schema Configuration"
+weight: 1
+slug: schema-configuration
+---
+
+Cortex uses a NoSQL Store to store its index and optionally an Object store to store its chunks. Cortex has overtime evolved its schema to be more optimal and better fit the use-cases that arised. Currently there are 10 schemas that are used in production. You can move from one schema to another if a new schema fits your purpose better, but you still need to configure Cortex to make sure it can read the old data in the old schemas.
+
+You can configure the schemas using a YAML config file, that you can point to using the `-schema-config-file` flag. It has the following YAML spec:
+
+```yaml
+configs: []<period_config>
+```
+
+Where `period_config` is
+```
+from: <Date>
+store: <string> // The index client to use.
+object_store: <string> // The object client to use. If none is specified, `store` is used for storing chunks as well.
+schema: <string> // The schema version to use. Valid ones are v1, v2, v3,..., v11
+index: <periodic_table_config>
+chunks: <periodic_table_config>
+```
+
+Where `periodic_table_config` is
+```
+prefix: <string> // The prefix to use for the tables.
+period: <time.Duration> // The period at which new tables are created and used. Typically 168h
```

Comment:
`The period at which new tables are created and used.` sounds a little confusing for a new user. Any other way to describe it? Maybe let us not correlate with when tables are created and used?

Reply:
Is it better now?

Reply:
Much better, just add a space in `theperiod`. Thanks!